tools/data-handler/src/rename.ts

```diff
@@ -74,18 +74,38 @@ export class Rename extends EventEmitter {
     const sortedCards = cards.sort((a, b) => sortCards(a, b));
 
     // Cannot do this parallel, since cards deeper in the hierarchy needs to be renamed first.
-    // First update all the attachments
+    // First update all the attachments and links.
     for (const card of sortedCards) {
       await this.updateCardAttachments(re, card);
+      await this.updateCardLinks(re, card);
     }
     // Then rename the cards
     for (const card of sortedCards) {
       await this.renameCard(re, card);
     }
   }
 
+  // Rename card links.
+  private async updateCardLinks(re: RegExp, card: Card) {
+    if (!Project.isTemplateCard(card)) {
```

Comment:
I would change this if to be opposite. 
if(Project.isTemplateCard(card)) {
  return;
}
rest of the code.

This way we don't have unnecessary nesting in the code.

Reply:
Sure, why not.